docs/console/design-your-projects/sidecars.md

```diff
@@ -31,24 +32,54 @@ By clicking on the Edit button inside the table, you can update the sidecar reso
 
-If you want to view the sidecar configuration in detail, use the **View sidecar configuration** button or click on the the sidecar name in the table row to go to its [detail page](/development_suite/api-console/api-design/microservice-containers.md#sidecar-detail) inside the microservice it is attached to.
+If you want to view the sidecar configuration in detail, use the **View sidecar configuration** button or click on the the sidecar
+name in the table row to go to its [detail page](/development_suite/api-console/api-design/microservice-containers.md#sidecar-detail) inside the microservice it is attached to.
 
 :::info
-Some sidecar configurations cannot be directly modified in this page. This is the case of [Special sidecars](/development_suite/api-console/api-design/microservice-containers.md#special-sidecars) that are managed by specific Console sections.
+Some sidecar configurations cannot be directly modified in this page.
+This is the case of [Special sidecars](/development_suite/api-console/api-design/microservice-containers.md#special-sidecars) that are managed by specific Console sections.
 :::
 
 #### Create sidecar
 
-You can add a new sidecar to your services by choosing an already existing sidecar configuration from the Marketplace or by specifying a docker image. To do this, select your best fitting solution from the **Create sidecar** dropdown.  
+You can add a new sidecar to your services by choosing an already existing sidecar configuration from the Marketplace or by specifying a docker image.
+To do this, select your best fitting solution from the **Create sidecar** dropdown.  
 From the creation form you can select multiple services to which the sidecar will be added.
 
+:::tip
+If you want to create a new instance of a sidecar you are already using in some microservice you can always [clone it](#clone-a-sidecar)!
+:::
+
 ![Create sidecar from docker image](img/create-sidecar.png)
 
 :::tip
-It is possible to define and publish sidecar resource definitions on the Marketplace. Find out more in the [Publish your sidecar to the Marketplace](/marketplace/add_to_marketplace/add_item_by_type/add_sidecar.md) guide.
+It is possible to define and publish sidecar resource definitions on the Marketplace.
+Find out more in the [Publish your sidecar to the Marketplace](/marketplace/add_to_marketplace/add_item_by_type/add_sidecar.md) guide.
 :::
 
-#### Delete sidecar
+#### Clone a sidecar
+
+Sometimes you may need to **replicate a sidecar configuration** for new microservices you have created,
+this is the scenario where you may need to **clone an existing sidecar** configuration into one or more existing microservices.
+
+To do so you can use the **_Add to another microservice_** microservice button.
+
+![Clone sidecar button](img/clone-sidecar-button.png)
+
+In the dialog, then select the microservices you wish to **copy the configuration from** and **one or more** microservices where the configuration
+should be **cloned to**.
```

Comment:
```suggestion
In the dialog, just select the microservice you wish to **copy the configuration from** and **one or more** microservices where the configuration should be **cloned to**.
```

Reply:
same, I've split some lines because they would fail lint validation for being too long. Even if the md is split the text will be correctly displayed